src/commonComponents/SchemaCard.tsx

```diff
@@ -45,9 +71,13 @@ const handleCheckboxChange = (checked: boolean, schemaData?: ISchemaData) => {
       if (!props.w3cSchema) {
         props.onClickCallback(props.schemaId, props.attributes, props.issuerDid, props.created)
       }
+      if (props.w3cSchema) {
+        props.onClickW3CCallback(props.schemaId, props.schemaName, props.version, props.issuerDid, props.attributes, props.created)
```

Comment:
If the parameter value is more than 2, create a separate object and pass the values as parameter

Reply:
Yes. Resolved.